docs/usage.rst

```diff
@@ -2,6 +2,6 @@
 Usage
 =====
 
-To use Data_models in a project::
```

Comment:
Ooops.

This is a relict from a Python template copy pasta haha.

The usage section should look different.
Has to be fixed in the template though and not in this PR.